selfdrive/controls/lib/model_parser.py

```diff
@@ -44,7 +44,7 @@ def update(self, v_ego, md):
     self.lane_width_certainty += 0.05 * (lr_prob - self.lane_width_certainty)
     current_lane_width = abs(l_poly[3] - r_poly[3])
     self.lane_width_estimate += 0.005 * (current_lane_width - self.lane_width_estimate)
-    speed_lane_width = interp(v_ego, [0., 31.], [3., 3.8])
+    speed_lane_width = interp(v_ego, [0., 31.], [2.8, 3.5])
```

Comment:
The Interstate Highway standards for the U.S. Interstate Highway System use a 12-foot (3.7 m) standard lane width, while narrower lanes are used on lower classification roads. In Europe, laws and road widths vary by country; the minimum widths of lanes are generally between 2.5 to 3.25 meters (8.2 to 10.7 ft).

So yeah, I think assuming slightly smaller lane lines by default will help for the average case and also for the case you described.